portfolio/src/main/webapp/script.js

```diff
@@ -218,6 +226,11 @@ function catCommand(commandArgument) {
 
   currentSection = newSection;
 
+  //TODO: Check if bugs when loading multiple times
+  if (currentSection == 'comments') {
+    loadComments();
+  }
```

Comment:
I'm currently checking if loading multiple times the comment section results in duplicated comments showing in the page.